python/python_direct/ops.cpp

```diff
@@ -49,6 +48,454 @@ Returns
 -------
 Val or TensorView
     The sum of the inputs.
+)")
+      NVFUSER_DIRECT_BINDING_BINARY_OP(
+          "atan2",
+          atan2,
+          R"(
+Element-wise arctangent of lhs/rhs choosing the quadrant.
+
+Parameters
+----------
+lhs : Val or TensorView
+    The y-coordinates.
+rhs : Val or TensorView
+    The x-coordinates.
+
+Returns
+-------
+Val or TensorView
+    The angles in radians between the positive x-axis and the rays to the points (x,y).
+)")
+          NVFUSER_DIRECT_BINDING_BINARY_OP(
+              "div",
+              div,
+              R"(
+Element-wise integer division.
+
+Parameters
+----------
+lhs : Val or TensorView
+    The dividend.
+rhs : Val or TensorView
+    The divisor.
+
+Returns
+-------
+Val or TensorView
+    The integer quotient.
+)")
+              NVFUSER_DIRECT_BINDING_BINARY_OP(
+                  "truediv",
+                  truediv,
+                  R"(
+Element-wise true (floating point) division.
+
+Parameters
+----------
+lhs : Val or TensorView
+    The dividend.
+rhs : Val or TensorView
+    The divisor.
+
+Returns
+-------
+Val or TensorView
+    The floating point quotient.
+)")
+                  NVFUSER_DIRECT_BINDING_BINARY_OP(
+                      "fmod",
+                      fmod,
+                      R"(
+Element-wise floating-point remainder.
```

Comment:
fmod is not remainder.

Reply:
I don't like their documentation neither... We don't need to explain math in our API, this is a fmod and we can just call it a floating-point mod as-is.

btw.
```
float x = -7.0;
float y = 4.0;
std::fmod(x, y); // -3.0
std::remainder(x, y); // 1
```